scripts/install_dev_env.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+# ********************************************************************
+# Complete development environment installation for Navitia.
+# After the installation, you'll be able to :
+# - Compile kraken
+# - Run all integration ad Unit tests
+# - Run docker (tyr + eitri) tests
+# - Run binaries to fill ed database (fusio2ed, osm2ed, ...)
+# - Run ed2nav binary to generate data.nav.lz4 form the ed database
+# - Run eitri (the docker method) to build data.nav.lz4 with your own NTFS and Osm files
+# ********************************************************************
+
+source ./dev_env/0_utils.sh
+source ./dev_env/1_shell_variables.sh
+source ./dev_env/2_install_packages.sh
+source ./dev_env/3_build.sh
+source ./dev_env/4_database_postgresql.sh
+source ./dev_env/5_python_env.sh
+
+
+# entry point
+
+DISTRIBUTION=`cat /etc/os-release | perl -ne 'print "$1\n" if /PRETTY_NAME="(.*?)"/'`
```

Comment:
Perl ?

Reply:
Add as dependency ?

Reply:
I don't think so, Perl is native on Debian/ubuntu. But why not a classical sed/awk ?

Reply:
No it is also native on Debian/Ubuntu. But you're right, it's better to keep it on Shell instead of adding a dependency to Perl.
Fixed